packages/react/src/domain/handles.ts

```diff
@@ -112,7 +122,8 @@ export type AuthenticatingMessage = {
   message: TextConfigKey;
 };
 export function getAuthenticatingMessage(
```

Comment:
Can we check if this function is being used anywhere else but in `authenticating.tsx`? If not, I would suggest moving it to the same file so it has a stronger context.

Reply:
You're right, it's only used in in `authenticating.tsx` I will move it there